packages/expo-firebase-polyfill/src/FirebasePolyfill.ts

```diff
@@ -0,0 +1,44 @@
+import { Platform } from 'react-native';
+
+import { btoa, atob } from './Base64';
+import { getRandomValues } from './Crypto';
+
+// Polyfill `btoa`
+// @ts-ignore
+if (!global.btoa) {
+  // @ts-ignore
+  global.btoa = btoa;
+}
```

Comment:
We should probably have a unified base64 package in expo so things like "no pad" work across modules. In that case we might consider moving this there.

Reply:
What do you mean by the "no pad" problem? Could you illustrate that?

Reply:
Like a package/documentation for reliably performing base64 encoding in a way that can emulate these different options: [`nsdatabase64encodingoptions`](https://developer.apple.com/documentation/foundation/nsdatabase64encodingoptions?language=objc)

Reply:
What other packages that we have right now would benefit from that?